packages/strapi-plugin-upload/services/Upload.js

```diff
@@ -171,6 +171,10 @@ module.exports = {
           return file;
         });
 
+        config.providerOptions = {
+          ...(details.folder && { folder: details.folder }),
```

Comment:
How can I add the `folder` option to the field?

Reply:
I imagine it here:

<img width="935" alt="Screenshot 2020-01-23 at 12 36 18" src="https://user-images.githubusercontent.com/1007051/72981445-0ccc3e00-3ddd-11ea-90e6-6606bd6e4415.png">